deployment/overview.md

```diff
@@ -52,7 +52,7 @@ The following articles provide information on the most common issues and questio
 
 * [CI, CD, Build Server]({%slug deployment-ci-cd-build-pc%}) - guidance on setting up continuous integration, continuous delivery and automated builds. Mostly related to using the Telerik NuGet packages as this is the only specific thing that we require.
 
-* [Telerik Private NuGet Feed]({%slug installation/nuget%}) - general guidance on setting up our private NuGet feed for local use
+* [Telerik NuGet Feed]({%slug installation/nuget%}) - general guidance on setting up our private NuGet feed for local use
```

Comment:
Consider switching to `&mdash;`, no spaces around, instead of a hyphen for the entire list. It improves readability and is usually the preferred punctuation for this type of info. More about this [here](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use).

Reply:
I personally dislike mdashes (actually the lack of spaces), so I will simply reword the text to avoid them.